packages/honeypot/http.go

```diff
@@ -51,9 +52,19 @@ func (h *httpHoneypot) Start() error {
 				},
 			},
 		}
+		vi := viper.New()
```

Comment:
You are reading the config on every http request. You can do it once (usually in main) and access the values the same way like you did. Viper will store it globally in their module.